data_safe_haven/commands/context.py

```diff
@@ -58,7 +58,7 @@ def add(
         str,
         typer.Option(
             help="The ID of an Azure group containing all administrators.",
-            callback=validate_aad_guid,
+            callback=typer_validate_aad_guid,
```

Comment:
This doesn't seem to work. I get a validation error. It seems it returns `None`. If I replace it with `validate_aad_guid`, then it works.

Reply:
Funnily enough I just found that myself working on #1674 :smile:.

Fixed in ba05dd7 and e1a0264

Reply:
I tried to fix it myself but put `return x` in the wrong place 😆